### PR TITLE
Add env vars to set custom home location

### PR DESCRIPTION
**Pull request by julianoes**
This adds the environment variables PX4_HOME_LAT, PX4_HOME_LON,
and PX4_HOME_ALT to specify a custom home/takeoff location.

@bkueng please review
@LorenzMeier objections?

### CODE REVIEW

**Comment by LorenzMeier** [comment]
@julianoes I think its a suitable stopgap solution. Can you quickly document that in the dev guide? Please use the WEB EDITOR! Not Github.

**Review by LorenzMeier — APPROVED** [review]

File: src/gazebo_mavlink_interface.cpp [hunk 1]
```diff
@@ -52,6 +57,24 @@ void GazeboMavlinkInterface::Load(physics::ModelPtr _model, sdf::ElementPtr _sdf
 
   world_ = model_->GetWorld();
 
+  // Use environment variables if set for home position.
+  const char *env_lat = std::getenv("PX4_HOME_LAT");
+  const char *env_lon = std::getenv("PX4_HOME_LON");
+  const char *env_alt = std::getenv("PX4_HOME_ALT");
+
+  if (env_lat) {
+    std::cout << "Home latitude is set to " << env_lat << "." << std::endl;
```
Comment (bkueng) [inline src/gazebo_mavlink_interface.cpp]: Gazebo uses `gzmsg <<`, `gzwarn`, etc. for output

**Comment by julianoes** [comment]
> Please use the WEB EDITOR! Not Github.

Huh, why?

File: src/gazebo_mavlink_interface.cpp (hunk 1 above)
Comment (julianoes) [reply src/gazebo_mavlink_interface.cpp]: Thanks, fixed in c3f44c3.

**Comment by LorenzMeier** [comment]
> Please use the WEB EDITOR! Not Github.
> Huh, why?

Because Gitbook is broken and the reference is the Gitbook web editor right now.

**Comment by julianoes** [comment]
Got it but not lucky editing it, even though I tried to authorize using github.

selection_072

**Pull request merged**